templates/docker_compose_postgres.yml

```diff
@@ -7,9 +7,9 @@ image: postgres:14
     environment:
       POSTGRES_USER: postgres
       POSTGRES_DB: relecloud
-      POSTGRES_HOST: db
       POSTGRES_PASSWORD: postgres
-      POSTGRES_PORT: 5432
```

Comment:
The postgres image doesn't use _HOST or _PORT env variables, so these were having no effect